docs: add URL to Log entry deletion page (#9119)
grafana/loki: docs/sources/operations/storage/retention.md

or
[GCS's documentation](https://cloud.google.com/storage/docs/managing-lifecycles).

> If you suddenly must delete ingested logs, you can delete old chunks in your object store. Note,

Reviewer: "Suddenly" just sounds odd here.
```suggestion
If you must delete ingested logs, you can delete old chunks in your object store. Note,
```

however, that this only deletes the log content and keeps the label index
intact; you will still be able to see related labels but will be unable to
retrieve the deleted log content.

For further details on the Table Manager internals, refer to the
[Table Manager]({{<relref "table-manager">}}) documentation.

> Alternatively, if the BoltDB Shipper is configured for the index store, you can enable [Log entry deletion]({{<relref "logs-deletion">}}) to delete log entries from a specific stream.

Reviewer: ```suggestion
Alternatively, if the BoltDB Shipper is configured for the index store, you can enable [Log entry deletion]({{<relref "logs-deletion.md">}}) to delete log entries from a specific stream.
```

or
[GCS's documentation](https://cloud.google.com/storage/docs/managing-lifecycles).

If you must delete ingested logs, you can delete old chunks in your object store. Note,
however, that this only deletes the log content and keeps the label index
intact; you will still be able to see related labels but will be unable to
retrieve the deleted log content.

For further details on the Table Manager internals, refer to the
[Table Manager](https://grafana.com/docs/loki/<LOKI_VERSION>/operations/storage/table-manager/) documentation.

> Alternatively, if the BoltDB Shipper is configured for the index store, you can enable [Log entry deletion]({{<relref "logs-deletion.md">}}) to delete log entries from a specific stream.

Reviewer: ```suggestion
Alternatively, if the BoltDB Shipper is configured for the index store, you can enable [Log entry deletion](https://grafana.com/docs/loki/<LOKI_VERSION>/operations/storage/logs-deletion/) to delete log entries from a specific stream.
```
